data/free worlds reconciliation.txt

```diff
@@ -1282,6 +1282,7 @@ mission "FW Pug 2C: Jump Drive"
 	on accept
 		outfit "Jump Drive" 1
 		set "fw given jump drive"
+		set "already had jump drive by FW Pug 2C"
```

Comment:
This seems pretty lengthy (and explicit, with the mission name reference) to me.

Reply:
What would you suggest?

Reply:
Simply "already had jump drive" might suffice.